pyproject.toml

```diff
@@ -11,6 +11,7 @@ python-dotenv = "^0.14.0"
 requests = "^2.25.1"
 pytest = "^6.2.4"
 selenium = "^3.141.0"
```

Comment:
Poetry lets you set some dependencies as "dev only", meaning that they aren't needed when running your app in production. `pytest` and `selenium` are good examples of this.

You can move them down this file into the `dev-dependencies` section below, and then in your Dockerfile you could have different `poetry install` steps for the production and development stages. For production use `poetry install --no-dev`